db/migrate/20230301083330_add_post_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Kindly delete the second migration between this file and [this one](https://github.com/soesandarwin2201/blog_with_ruby_on_rails/pull/1/files#diff-535ba548a227ea5416a98368a4fe6594ce3db79961350a8bb6ca410dea94db8c)